### ISSUE TITLE
Header content type doesnt change

### ISSUE THREAD

**Issue by fponia** [AshV/AshV.WebApiTester.XTB#4]
Hello i wanted to do a Batch Request Operation in the webApi Launcher but when i changed the Content Type Header from the default one to multipart/mixed it doesnt change it in the background ( found the issue via Fiddler) Thx

**Comment by AshV** [comment]
Hi, I just wanted to quickly confirm, Have you checked that checked box before header name before triggering request?

![image](https://user-images.githubusercontent.com/7019016/132251651-d5452870-6993-47eb-af82-a9a85948bbb9.png)

**Comment by fponia** [comment]
Yes it was checked. Btw I also tried to uncheck the default one and create
my own Content Type.
The same problem:
[image: image.png]

**Comment by JurgenPostelmans** [comment]
I have the same issue with the Content-Type header. Fiddler indeed shows that the wrong content-type header is transmitted...
